pages/_app.js

```diff
@@ -1,26 +1,30 @@
 import React from "react";
 import App from "next/app";
+import Router from "next/router";
+import { GoogleReCaptchaProvider } from "react-google-recaptcha-v3";
 import "@styles/variables.scss";
 import "@styles/font.scss";
 import "@styles/style.scss";
 import "@styles/loader.scss";
 
 import { AnimatePresence } from "framer-motion";
-import AppContextProvider from "@helpers/appContext";
+import { pageview } from "@helpers/googleAnalytics";
+
+Router.events.on("routeChangeComplete", (url) => pageview(url));
 
 class MyApp extends App {
   render() {
     const { Component, pageProps, router } = this.props;
     // <Component/> returns the component it self
     // pageProps returns the props you use to in that component. It can be any data
-    // router.route returns the route your component lives on. So in our case it will be '/' or '/products/[id]'
+    // router.route returns the route your component lives on.
     // exitBeforeEnter: AnimatePresence will only render one component at a time. The exiting component will finished its exit animation before the entering component is rendered
     return (
-      <AppContextProvider>
+      <GoogleReCaptchaProvider reCaptchaKey="6LdfR-oUAAAAAHZ7qMBCS85ZoALGOUSAhVEoUFuZ">
```

Comment:
Can set `reCaptchaKey` value as an env variable or in a config file?

Reply:
Maaybe? No necesitamos realmente q sea oculto asi que por eso lo dejé acá 🤔 
Podria ser, voy a cambiarlo a Env vars

Reply:
Vale, igual para mi esto no es tan solo por temas de seguridad, también es por legibilidad y orden en el código.